Index: hwloc/doc/Makefile.am
===================================================================
--- hwloc.orig/doc/Makefile.am	2009-11-19 02:21:26.000000000 +0100
+++ hwloc/doc/Makefile.am	2009-11-19 02:21:33.000000000 +0100
@@ -345,13 +345,11 @@ endif
 # Make sure that the documentation example works
 #
 
-if BUILD_DOXYGEN
 TESTS = hwloc-hello
 check_PROGRAMS = $(TESTS)
 
 hwloc_hello_LDADD = $(top_builddir)/src/libhwloc.la
 hwloc_hello_DEPENDENCIES = $(hwloc_hello_LDADD)
-endif
 
 #
 # Only remove the actual generated documentation files for maintainers
Index: hwloc/doc/Makefile.in
===================================================================
--- hwloc.orig/doc/Makefile.in	2009-11-19 02:21:40.000000000 +0100
+++ hwloc/doc/Makefile.in	2009-11-19 02:22:09.000000000 +0100
@@ -43,8 +43,8 @@ target_triplet = @target@
 DIST_COMMON = $(am__dist_pdf_DATA_DIST) $(srcdir)/Makefile.am \
 	$(srcdir)/Makefile.in $(srcdir)/doxygen-config.cfg.in \
 	$(top_srcdir)/hwloc.am
-@BUILD_DOXYGEN_TRUE@TESTS = hwloc-hello$(EXEEXT)
-@BUILD_DOXYGEN_TRUE@check_PROGRAMS = $(am__EXEEXT_1)
+TESTS = hwloc-hello$(EXEEXT)
+check_PROGRAMS = $(am__EXEEXT_1)
 subdir = doc
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
 am__aclocal_m4_deps = $(top_srcdir)/config/hwloc.m4 \
@@ -62,7 +62,7 @@ CONFIG_HEADER = $(top_builddir)/include/
 	$(top_builddir)/include/hwloc/config.h
 CONFIG_CLEAN_FILES = doxygen-config.cfg
 CONFIG_CLEAN_VPATH_FILES =
-@BUILD_DOXYGEN_TRUE@am__EXEEXT_1 = hwloc-hello$(EXEEXT)
+am__EXEEXT_1 = hwloc-hello$(EXEEXT)
 hwloc_hello_SOURCES = hwloc-hello.c
 hwloc_hello_OBJECTS = hwloc-hello.$(OBJEXT)
 DEFAULT_INCLUDES = -I.@am__isrc@ -I$(top_builddir)/include/private -I$(top_builddir)/include/hwloc
@@ -476,8 +476,8 @@ dox_inputs = $(DOX_CONFIG) \
 @INSTALL_DOXYGEN_TRUE@        $(DOX_MAN_DIR)/man3/hwlocality_traversal.3 \
 @INSTALL_DOXYGEN_TRUE@        $(DOX_MAN_DIR)/man3/hwlocality_types.3
 
-@BUILD_DOXYGEN_TRUE@hwloc_hello_LDADD = $(top_builddir)/src/libhwloc.la
-@BUILD_DOXYGEN_TRUE@hwloc_hello_DEPENDENCIES = $(hwloc_hello_LDADD)
+hwloc_hello_LDADD = $(top_builddir)/src/libhwloc.la
+hwloc_hello_DEPENDENCIES = $(hwloc_hello_LDADD)
 
 #
 # Only remove the actual generated documentation files for maintainers
